### PR TITLE
Release next production.

### PR DESCRIPTION
**Pull request by Anduin2017**

### CODE REVIEW

File: BLACK_LIST [hunk 1]
```diff
@@ -26,3 +26,7 @@ aboluowang.com
 secretchina.com
 soundofhope.org
 newsgni.com
+cnn.com
+newsweek.com
+pornhub.com
+x-videos.com
```
Comment (MESOLONELY) [inline BLACK_LIST:32]: I don't like this black list

**Review by MESOLONELY — APPROVED** [review]

File: tests/SDKTools.Tests/AttributesTests/ValidFolderNameTests.cs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        [DataRow("aaaatxt")]
+        [DataRow("asdfasdfadfasdfasdfadf___^&(ad")]
+        public void PassingTests(object inputValue)
+        {
+            Assert.AreEqual(ValidationResult.Success, _validator.TestEntry(inputValue));
+        }
+
+        [TestMethod]
+        [DataRow("asdfasdfadfasdfasdfadf___^&(ad*")]
+        [DataRow(@"
```
Comment (ganlvtech) [inline tests/SDKTools.Tests/AttributesTests/ValidFolderNameTests.cs:29]: Test data here is just `0d 0a`(`\r\n`).

Maybe this test case cannot cover https://github.com/AiursoftWeb/AiurDrive/issues/4.

**Review by ganlvtech — CHANGES_REQUESTED** [review]
Test case needs change.

File: tests/SDKTools.Tests/AttributesTests/ValidFolderNameTests.cs (hunk 2 above)
Comment (Anduin2017) [reply tests/SDKTools.Tests/AttributesTests/ValidFolderNameTests.cs:32]: #4. Using a new test case to cover that.

**Review by ganlvtech — APPROVED** [review]

**Pull request merged**